frontend/src/components/object/ObjectVersion.vue

```diff
@@ -0,0 +1,175 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+import { onMounted, ref, watch } from 'vue';
+
+import {
+  DeleteObjectButton,
+  DownloadObjectButton
+} from '@/components/object';
+import { Button, Column, DataTable, useToast } from '@/lib/primevue';
+import { useAppStore, useAuthStore, usePermissionStore, useUserStore, useVersionStore } from '@/store';
+import { Permissions, RouteNames } from '@/utils/constants';
+import { ButtonMode } from '@/utils/enums';
+import { formatDateLong } from '@/utils/formatters';
+
+import type { Ref } from 'vue';
+import type { User, Version } from '@/types';
+
+type VersionDataSource = {
+  createdByName?: string;
+} & Version;
+
+// Props
+type Props = {
+  bucketId: string;
+  objectId: string;
+  versionId: string;
+};
+
+const props = withDefaults(defineProps<Props>(), {});
+
+// Store
+const permissionStore = usePermissionStore();
+const userStore = useUserStore();
+const versionStore = useVersionStore();
+const { getUserId } = storeToRefs(useAuthStore());
+const { getUserSearch } = storeToRefs(userStore);
+
+// State
+const tableData: Ref<Array<VersionDataSource>> = ref([]);
+
+// Actions
+const toast = useToast();
+
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+const showInfo = async (id: string) => {
+  // emit('show-object-info', id);
+};
```

Comment:
Do we still need this function at all if it's effectively a no-op?

Reply:
Leaving in for now. The button that even calls this is disabled, but I believe I will be making use of this in the next PR for object updating.